src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Builds.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Immutable;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    public class ScenarioTests_Builds : MaestroScenarioTestBase
+    {
+        private string repoUrl;
```

Comment:
Why is this a field?  If there is every more than one test here, it's going to go really badly... and if there isn't... well, then why not just a local variable?

Reply:
This will also probably bite us in a lot of other tests.

Reply:
I can make the InitializeAsync & DisposeAsync run before & after each test, but when it was set up that way it was causing problems with the .Net install when run in parallel. I'll experiment a bit and see how much I can isolate.

Reply:
Yea, that was happening because they are sharing the _parameters field. That is wrong.

Every test needs a new TestParameters.